modules/core/src/main/scala/io/chrisdavenport/mules/MemoryCache.scala

```diff
@@ -280,6 +280,27 @@ object MemoryCache {
     Resource.make(runExpiration(memoryCache).start)(_.cancel).void
   }
 
+  /**
+    * Cache construction is Synchronous
+    * 
+    * Otherwise a copy paste of {@link #ofSingleImmutableMap() ofSingleImmutableMap}
+    * 
+    **/
+  def ofSingleImmutableMap2[G[_] : Sync, F[_]: Async, K, V](
```

Comment:
We typically use `in` to denote these dual effect constructors. Could this be `inSingleImmutableMap`? I think it matches other semantics better.

Reply:
Yes - of course...